src/index.js

```diff
@@ -510,10 +510,23 @@ var VERSION = '__VERSION__';
  * braintree.dropin.create({
  *   authorization: 'CLIENT_AUTHORIZATION',
  *   container: '#dropin-container',
- *   threeDSecure: {
- *     amount: '10.00'
- *   }
- * }, callback);
+ *   threeDSecure: true
+ * }, function (err, dropinInstance) {
+ *   // setup payment button
+ *   btn.addEventListener('click', function (e) {
+ *     e.preventDefault();
+ *
+ *     dropinInstance.requestPaymentMethod(|
+ *       threeDSecure: {
+ *         amount: '100.00',
+ *        // additional 3ds params
```

Comment:
Add one more space to indent here.

Also, can we include some more of these additional params instead of just having the comment mention them?